gcc/d/decl.cc

```diff
@@ -1108,8 +1120,9 @@ get_symbol_decl (Declaration *decl)
       /* Vector array operations are always compiler generated.  */
       if (fd->isArrayOp)
 	{
+	  TREE_PUBLIC (decl->csym) = 1;
 	  DECL_ARTIFICIAL (decl->csym) = 1;
-	  D_DECL_ONE_ONLY (decl->csym) = 1;
+	  d_comdat_linkage (decl->csym);
```

Comment:
Inlined compiler-generated arrayops are discarded. (i.e: one less symbol in the resultant binary).